apps/dialtone-documentation/docs/design/colors/dataviz/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Data visualization
+description: Data visualization is crucial for clear communication, but inconsistent color usage can hinder comprehension and create visual noise.
```

Comment:
Let's avoid "Data Visualization", and keep it purely about "these are the available Chart Colors."

That said, we definitely should have a design-specific doc page in the Design section that covers all design considerations for data viz beyond just color. But we're not ready for that, and just need to get Chart Colors published.